plugins/search/src/components/Filters/Filters.tsx

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import {
+  makeStyles,
+  Typography,
+  Divider,
+  Card,
+  CardHeader,
+  Button,
+  CardContent,
+  Select,
+  Checkbox,
+  FormControlLabel,
+  MenuItem,
+} from '@material-ui/core';
+
+const useStyles = makeStyles({
+  filters: {
+    background: 'transparent',
+    boxShadow: '0px 0px 0px 0px',
```

Comment:
I use the Card component from material ui because I wanted to use its card header etc which works pretty well for this use case. But that component has box shadow as default, so deleted that for the filters box to make it more aligned with sketches from Kat.

Reply:
I see, maybe a slight abuse of the card but it's OK.